### PR TITLE
Docs: test keyword frontmatter

### PR DESCRIPTION
**Pull request by alexfornuto**
This PR adds a `keywords` frontmatter field to a test page. This accompanies https://github.com/gravitational/docs/pull/239 from the docs repo, which adds the content of this field to a keyword meta tag.

These values will also be used to optimize search.

### CODE REVIEW

**Review by ptgott — APPROVED** [review]
We should think about limiting the use of keywords to an allowlist so we don't have too many keywords that correspond to only a single docs page

**Comment by alexfornuto** [comment]
> We should think about limiting the use of keywords to an allowlist so we don't have too many keywords that correspond to only a single docs page

Fair enough. For now I just want to confirm that the docs PR to add meta tags works, and that Algolia will scrape the keywords. I chose this docs to test with because it's so big it's been excluded from crawling and doesn't show up in search results, so once that's fixed it could use a leg up to be more visible.

File: docs/pages/reference/config.mdx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Teleport Configuration Reference
 description: The detailed guide and reference documentation for configuring Teleport for SSH and Kubernetes access.
+keywords: [config, file, reference, yaml, etc]
```
Comment (alexfornuto) [inline docs/pages/reference/config.mdx:4]: Example suggestion. Ignore me, gonna delete this.

```suggestion
keywords: [config, file, reference, yaml, etc, aardvark]
```

File: docs/pages/reference/config.mdx [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
 ---
 
 Teleport uses the YAML file format for configuration. A full configuration
```
Comment (alexfornuto) [inline docs/pages/reference/config.mdx:7]: Example suggestion. Ignore me, gonna delete this.

```suggestion
Teleport uses the a combination of go, rust, ini, conf, and apache config file format for configuration. A full configuration
```

**Review by stevenGravy — APPROVED** [review]

**Pull request merged**